log/go.mod

```diff
@@ -6,16 +6,20 @@ require (
 	github.com/bytedance/sonic v1.12.3
 	github.com/pkg/errors v0.9.1
 	github.com/rs/zerolog v1.33.0
+	github.com/stretchr/testify v1.8.1
```

Comment:
Can we not re-add testify?

Reply:
Sure, let me push another change to go back to standard library testing.